index.html

```diff
@@ -0,0 +1,144 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <header id="header">
+  <img src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSt_S0TvjguVUdnKEheN1MgkcuIVWAlTOVWoA&usqp=CAU" alt="" id="header-img">
+  <nav id="nav-bar">
+    <li class="nav-item"> 
+      <a href="#services" class="nav-link">Services</a>
+    </li>
+     <li class="nav-item"> 
+      <a href="#how-it-works" class="nav-link">How It Works</a>
+    </li>
+     <li class="nav-item">
+      <a href="#packages" class="nav-link">Packages</a>
+    </li>
+  </nav>
```

Comment:
It is a good practice to put the nav in <header> </header>

Reply:
- [ ] Awesome observation! can you please rephrase your comment politely as a suggestion 💁‍♂️ and provide reason(s) for the required changes? It may be obvious to you and not so for others. 👍